src/transformers/modeling_utils.py

```diff
@@ -1896,9 +1896,11 @@ def save_pretrained(
                 Whether or not to push your model to the Hugging Face model hub after saving it. You can specify the
                 repository you want to push to with `repo_id` (will default to the name of `save_directory` in your
                 namespace).
-            max_shard_size (`int` or `str`, *optional*, defaults to `"10GB"`):
+            max_shard_size (`int` or `str`, *optional*, defaults to `"2GB"`):
                 The maximum size for a checkpoint before being sharded. Checkpoints shard will then be each of size
                 lower than this size. If expressed as a string, needs to be digits followed by a unit (like `"5MB"`).
+                We default it to 2GB in order for models to be able to run easily on free-tier google colab instances
+                without CPU OOM issues.
```

Comment:
maybe 5GB would be a better trade off no?

Reply:
Yeah makes sense